src/DDLQueryBuilder.php

```diff
@@ -173,21 +175,18 @@ public function renameColumn(string $table, string $oldName, string $newName): s
      */
     public function getColumnDefinition(string $table, string $column): string
     {
-        $result = '';
         $sql = $this->schema->getTableSchema($table)?->getCreateSql();
 
         if (empty($sql)) {
-            return '';
+            throw new InvalidArgumentException("Table \"$table\" does not exist.");
```

Comment:
It's not psalm fix. This change of behavior. Please move that to other PR.

Reply:
The line in the change log is added